common/djangoapps/student/management/tests/test_bulk_user_activate.py

```diff
@@ -0,0 +1,61 @@
+"""Tests for the bulk_user_activate command."""
+
+from django.core.management import call_command
+
+from django.contrib.auth.models import User
+from django.test import TestCase
+
+from student.models import get_users_by_email
```

Comment:
get_users_by_email is different from what's being done in the command file. I think there is utility in having this as a helper function in models, or somewhere, in the event it becomes useful in another context. @kluo did you ask Tony to do away with that function?

Reply:
I figured it was a pretty straightforward django filter, but I'm okay with keeping it